### PR TITLE
Improve resource autofocus

### PR DESCRIPTION
**Pull request by jcenturion**
I removed the uses of  ```$anchorScroll();```and ```$location.hash``` for method autofocus since the generated fragments were collapsing with another apps.

### CODE REVIEW

File: dist/scripts/api-console.js [hunk 1]
```diff
@@ -517,8 +517,12 @@
             jQuery($this).addClass('raml-console-is-active');
             $scope.showPanel = true;
 
-            $location.hash(hash);
-            $anchorScroll();
+            setTimeout(function () {
```
Comment (AlejoFernandez) [inline dist/scripts/api-console.js]: Can't we use `$timeout` instead? or do we need to run this outside of an angular digest loop? If that's the case let's document in the code the reason why are we using `setTimeout`

File: src/app/directives/root-documentation.js [hunk 2]
```diff
@@ -31,7 +31,11 @@
 
           $el[!$scope.documentationEnabled ? 'removeClass' : 'addClass']('raml-console-documentation-active');
 
-          $location.hash($scope.generateDocId(section));
+          setTimeout(function () {
+            jQuery('html, body').animate({
```
Comment (AlejoFernandez) [inline src/app/directives/root-documentation.js]: Will it be possible to fix this without using jQuery?
There is a `$animate` service in angular, although I'm not sure if it works exactly as jQuery's version.
Found an example here: http://stackoverflow.com/questions/24850066/replace-jquery-animation-with-built-in-angular-animation

File: dist/scripts/api-console.js (hunk 1 above)
Comment (jcenturion) [inline dist/scripts/api-console.js]: Done.

**Comment by AlejoFernandez** [comment]
:+1: LGTM

**Pull request merged**